sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -54,6 +54,26 @@ static const struct snd_soc_acpi_adr_device rt711_1_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt711_1_aif1_adr[] = {
+	{
+		.adr = 0x000120025D071100,
+		.dai_name = "rt711-aif1",
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
+		.name_prefix = "rt711"
+	}
+};
+
+static const struct snd_soc_acpi_adr_device rt711_1_aif2_adr[] = {
+	{
+		.adr = 0x000120025D071100,
+		.dai_name = "rt711-aif2",
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
+		.name_prefix = "rt711"
```

Comment:
same comment as earlier, it feels like the dai_name is a property of the endpoint. So in this case you could have 2 endpoints with different descriptors for headset and mic.